oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -307,10 +307,22 @@ private boolean indexProperty(String path,
     /**
      * Returns {@code true} if nodeScopeIndex full text values need to be indexed
      */
+    @Deprecated
```

Comment:
no need to deprecate. We can safely remove this since I have introduced it last week.